packages/dev/core/src/Meshes/mesh.vertexData.ts

```diff
@@ -766,7 +766,8 @@ export class VertexData {
     ): Nullable<FloatArray> {
         const nonNullOthers = others.filter((other): other is [element: FloatArray, transform?: Matrix] => other[0] !== null && other[0] !== undefined);

```

Comment:
My understanding of mergeMeshes is that we can only merge meshes when vertex buffer attributes match, but the way this code is written seems to account for cases where certain elements are only defined for certain vertex buffers.  Does anyone know if this filter is actually needed or do we expect that nonNullOthers  should always be equal to others?